manimlib/mobject/svg/code_mobject.py

```diff
@@ -245,6 +333,12 @@ def gen_code_json(self):
         # print(self.code_json)
 
     def correct_non_span(self, line_str):
+        """Function put text color to those strings that don't have one according to background_color of displayed code.
+        Arguments
+        ---------
+        line_str : :class:`str`
+            Takes a string to put color to it according to background_color of displayed code.
+        """

```

Comment:
please document this return value
```suggestion

        Returns
        --------
        :class:`str`
        """
```